WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -17,7 +17,7 @@
         <% else %>
           <%= icon('hourglass-start', title: "Starts in #{ pluralize((competition.start_date - Date.today).to_i, "day") }", data: { toggle: "tooltip" }) %>
         <% end %>
-        <%= wca_date_range(competition.start_date, competition.end_date)  %>
+        <%= competition.start_date && competition.end_date ? wca_date_range(competition.start_date, competition.end_date) : "Unscheduled" %>
```

Comment:
Related to this. Just found out a funny issue. If you create a competition it hasn't a start and end dates (which is the correct behavior) but you can make it visible immediately. One of the results is that on the index page you can select `Past from 0` year (there's no exception so it's filtered out in the controller probably by chance). @jfly is it because we rely on people that they won't make such a incomplete competition visible?
So I think we don't need this `"Unscheduled"`. If there's a really danger that such a comp may became visible, then we should rather prevent from this directly.